app/models/diy_intake.rb

```diff
@@ -42,7 +42,7 @@ def issue_token
   end
 
   def start_filing_url
-    Rails.application.routes.url_helpers.diy_start_filing_url(token)
+    Rails.application.routes.url_helpers.diy_start_filing_url(:token => token)
```

Comment:
fyi, this is a common thing to watch out for when adding a locale scope to the app to support i18n. the first argument to paths is now always locale, so if you want to pass another arg like the token here or an id, it needs to be explicitly named